From 9207ee49b63f6cce44148bf97d5bd02d1ca540e2 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Tue, 11 Sep 2018 15:10:04 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808 fix pincrl error

1. fix emmc_clkout/emmc_rstn to gpio1 RK_PB1/RK_PB3
2. fix sdmmc1_clk to gpio4 RK_PA7
3. fix to pull_up_8ma for spi_mosi_hs pins

Change-Id: Ib67d0c6ab485f38e282f1b6e4cfba16b73da1df5
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index c5b89ee1fc02..4d17065b39eb 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -965,13 +965,13 @@
 			emmc_clk: emmc-clk {
 				rockchip,pins =
 					/* emmc_clkout */
-					<1 9 1 &pcfg_pull_none>;
+					<1 RK_PB1 1 &pcfg_pull_none>;
 			};
 
 			emmc_rstnout: emmc-rstnout {
 				rockchip,pins =
 					/* emmc_rstn */
-					<1 11 1 &pcfg_pull_none>;
+					<1 RK_PB3 1 &pcfg_pull_none>;
 			};
 
 			emmc_bus8: emmc-bus8 {
@@ -1383,7 +1383,7 @@
 
 			sdmmc1_clk: sdmmc1-clk {
 				rockchip,pins =
-					<4 RK_PB1 1 &pcfg_pull_none>;
+					<4 RK_PA7 1 &pcfg_pull_none>;
 			};
 		};
 
@@ -1509,27 +1509,27 @@
 
 			spi1m1_clk_hs: spi1m1-clk-hs {
 				rockchip,pins =
-					<3 RK_PC7 3 &pcfg_pull_none>;
+					<3 RK_PC7 3 &pcfg_pull_up_8ma>;
 			};
 
 			spi1m1_mosi_hs: spi1m1-mosi-hs {
 				rockchip,pins =
-					<3 RK_PD0 3 &pcfg_pull_none>;
+					<3 RK_PD0 3 &pcfg_pull_up_8ma>;
 			};
 
 			spi1m1_csn0_hs: spi1m1-csn0-hs {
 				rockchip,pins =
-					<3 RK_PD1 3 &pcfg_pull_none>;
+					<3 RK_PD1 3 &pcfg_pull_up_8ma>;
 			};
 
 			spi1m1_miso_hs: spi1m1-miso-hs {
 				rockchip,pins =
-					<3 RK_PD2 3 &pcfg_pull_none>;
+					<3 RK_PD2 3 &pcfg_pull_up_8ma>;
 			};
 
 			spi1m1_csn1_hs: spi1m1-csn1-hs {
 				rockchip,pins =
-					<3 RK_PD3 3 &pcfg_pull_none>;
+					<3 RK_PD3 3 &pcfg_pull_up_8ma>;
 			};
 		};
 
@@ -1598,22 +1598,22 @@
 
 			spi2m1_miso_hs: spi2m1-miso-hs {
 				rockchip,pins =
-					<2 RK_PA4 3 &pcfg_pull_none>;
+					<2 RK_PA4 3 &pcfg_pull_up_8ma>;
 			};
 
 			spi2m1_clk_hs: spi2m1-clk-hs {
 				rockchip,pins =
-					<2 RK_PA5 3 &pcfg_pull_none>;
+					<2 RK_PA5 3 &pcfg_pull_up_8ma>;
 			};
 
 			spi2m1_mosi_hs: spi2m1-mosi-hs {
 				rockchip,pins =
-					<2 RK_PA6 3 &pcfg_pull_none>;
+					<2 RK_PA6 3 &pcfg_pull_up_8ma>;
 			};
 
 			spi2m1_csn_hs: spi2m1-csn-hs {
 				rockchip,pins =
-					<2 RK_PA7 3 &pcfg_pull_none>;
+					<2 RK_PA7 3 &pcfg_pull_up_8ma>;
 			};
 		};
 
-- 
2.35.3

